Alicerce.podspec

```diff
@@ -26,6 +26,4 @@ Pod::Spec.new do |s|
 
     s.frameworks    = [ 'Foundation', 'UIKit', 'CoreData', 'Security' ]
     s.dependency 'Result', '~> 4.0'
-
-    s.script_phase  = { :name => 'CommonCrypto', :script => '${PODS_TARGET_SRCROOT}/install_common_crypto.sh', :execution_position => :before_compile }
```

Comment:
We should also remove the `s.preserve_paths` line, and delete the install_common_crypto.sh file 🗑